.gitignore

```diff
@@ -1 +1,13 @@
 /api-raml/
+
+test/__pycache__/
+
+venv/
+
+elife_api_validator/__pycache__/
```

Comment:
I wonder if some of these paths should have a leading `/` to specify they are only valid in the root:
```
/elife_api_validator/something
```
but then, you should be able to just use
```
__pycache__
```
which should catch them everywhere. If a file is named `__pycache__` rather than a directory we have bigger problems

Reply:
Now done. @thewilkybarkid advised me in https://github.com/elifesciences/api-validator-python/pull/3#discussion_r146820119